templates/frontend/objects/article_summary.tpl

```diff
@@ -24,7 +24,7 @@
 	{if $article->getLocalizedCoverImage()}
 		<div class="cover">
 			<a {if $journal}href="{url journal=$journal->getPath() page="article" op="view" path=$articlePath}"{else}href="{url page="article" op="view" path=$articlePath}"{/if} class="file">
-				<img src="{$article->getLocalizedCoverImageUrl()|escape}"{if $article->getLocalizedCoverImageAltText() != ''} alt="{$article->getLocalizedCoverImageAltText()|escape}"{else} alt="{translate key="article.coverPage.altText"}"{/if}>
+				<img src="{$article->getLocalizedCoverImageUrl()|escape}"{if $article->getLocalizedCoverImageAltText() != ''} alt="{$article->getLocalizedCoverImageAltText()|default:'null'}"{else} alt="{translate key="article.coverPage.altText"|default:'null'}"{/if}>
```

Comment:
Just following up on this comment:

> Looking at this further, I notice that here there is backup alt text if the alt text is missing. It's the locale key article.coverPage.altText which comes out at "Cover Image".
> 
> That's probably not a good alt text since it doesn't describe the image in any way. When there is no alt text, let's fall back to null.

So in this case, I think we can get rid of the `{else}` statement entirely. The line would be like:

```
<img src="{$article->getLocalizedCoverImageUrl()|escape}" alt="{$article->getLocalizedCoverImageAltText()|default:'null'}">
```